torch/_torch_docs.py

```diff
@@ -469,6 +469,35 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.as_strided,
+           r"""
+as_strided(input, size, stride, storage_offset=0) -> Tensor
+
+Create a `torch.Tensor` from an existing `torch.Tensor` :attr:`input` with
```

Comment:
I think it is important to mention that this creates a view of an existing tensor. :)

Reply:
You are right. Will ``Create a view of an existing torch.Tensor ...`` be better?

Reply:
Yes, that sounds great! Thank you!

Reply:
Also, could you also add an entry to the method version of it at https://github.com/pytorch/pytorch/blob/f2f80744be347fab6921fc55181b55dc86c7e07b/torch/_tensor_docs.py? It should be easy and could just refer to `torch.as_strided`.

Reply:
Actually I think `torch.as_strided` probably should not exist. Only the method version should, similar to `resize`. 

cc @gchanan who probably should have better opinion on this.

Reply:
I think it's fine, `torch.as_tensor` exists (because it needs to), so other `as_` functions seem like fair game.

Reply:
Cool. I will update both soon.